test/core/keyTests.ts

```diff
@@ -0,0 +1,45 @@
+﻿///<reference path="../testReference.ts" />
+
+describe("DatasetKey", () => {
+  it("is passed to Dataset", () => {
+    let ds: Plottable.Dataset = new Plottable.Dataset();
```

Comment:
Explicit typing is mainly unnecessary if Typescript can infer the type.

In this scenario, you are setting the variable to `new Plottable.Dataset()` (which is of type `Plottable.Dataset).  Thus, the variable can be inferred to be of type `Plottable.Dataset`